cf_units/tests/__init__.py

```diff
@@ -1,5 +1,4 @@
 # Copyright cf-units contributors
-#
-# This file is part of cf-units and is released under the LGPL license.
-# See COPYING and COPYING.LESSER in the root of the repository for full
-# licensing details.
+
```

Comment:
I think, according to the license-headers test, this line should have a "#", not just blank.
```suggestion
#
```